### ISSUE TITLE
Full game list of a user

### ISSUE THREAD

**Issue by veloce** [lichess-org/mobile#454]
For now we're only displaying the last 10 recent games.

A "More" button should open another screen with a lazy loading, infinite list of games.

On top of the list, an horizontal list of filters should be displayed (like in the notifications screen of GitHub app). Filters would include: "rated", "perf type", "bookmarks", etc. (list non exhaustive).

Use this endpoint: https://lichess.org/api#tag/Games/operation/apiGamesUser

(All filters are not yet available in the endpoint)

List items should be slidable to uncover actions:
- bookmark/unbookmark

Long press should open the context menu.

TODO: wireframe

**Comment by ijm8710** [comment]
Piece color and game type played (variant) should also probably be filtering options too

**Comment by ijm8710** [comment]
I agree with your first part Julien, but I still think bookmark as a game-list filter should be added eventually and is still not present.

Especially since the detailed view that archived games provides gives you more to sift through if you have a lot of bookmarks to scroll thru than the access point from your profile
